frontend/components/common/TagGroup.js

```diff
@@ -40,14 +52,14 @@ export const TagGroup = ({ tags = [], selectedTags = [], updateTag }) => {
             updateTag(
               {
                 value: tag.id,
-                label: tag.name,
+                label: tag.name || tag.label,
                 name: 'Type',
               },
               'Type'
             )
```

Comment:
Do we need updateTag? Did we make a mistake with https://github.com/pennlabs/pennclubs/pull/76?

Reply:
We actually don't need updateTag here, I just added the extra statement because it seemed like a reasonable alternative in case thee backend uses `label` instead of `name`. I will remove this.